play/PlayerRules/CharacterCreation/05_ArmourList.txt

```diff
@@ -88,12 +71,67 @@ this by providing 'Hard Points'. Hard Points are special joints and fittings tha
 make it easy to attach equipment. When you purchase equipment, it will list how
 many Hard Points it consumes. You can't attach more equipment than you have available
 Hard Point space.
+	Hard Point Equipment/Shields/Cloaking often come in two sizes, "Primary Modules" and 
+"Secondary Modules". After you purchase and install a Primary Module, you can also purchase
+and install a Secondary Module(s) that augments the benefits of the Primary Module. The 
+benefits are listed in the description of a given module. They do not mix and match,
+Secondary Modules only work for the Primary Module they are designed for.
+	Some modules have a "____ Skill Requirement". In order to operate the module, you
+must have the required amount of the given skill.
+
+
+    =====================================
+        EVA SUITS AND FLIGHT HELMETS
+    =====================================
+These additions may be worn with or added to armour. Environment 
+(EVA) suits are air- and water-tight when combined with an EVA or flight 
+helmet. EVA suits are bulky, DEX skill checks have a -10% penalty and movement
+is reduced in RP. If the suit is hit more times than its "Suit Integrity"
+rating, it will no longer be able to maintain air pressure (player health 
+has to be damaged in order for "Suit Integrity" to count a hit). Suits 
+traditionally come in white, yellow or red.
+
+	=====   LEVEL 1 REQUIREMENT   =====
+	
+Basic Tactical Helmet $100
+Tactical and Combat helmets can hold NVGs, lights, radios, etc.
+Protects the head from blunt trauma (mostly RP effects).
+
+Basic Flight Helmet $240
+A simple flight helmet. does not provide full seal, but can be paired with 
+air- and space-craft external oxygen systems. Protects the head from blunt
+trauma (mostly RP effects).
+
+Environment Helmet $350
+Provides a full environmental seal. Protects the head from blunt 
+trauma (mostly RP effects).
+
+-------------------------------------------------
+
+	=====   LEVEL 1 REQUIREMENT   =====
+	
+Basic Flight-Suit  $800
+Basic Environment Suit, offers no ballistic protection. When shot, 
+the suit will decompress or compress, as necessary to maintain
+air pressure.
+Suit integrity: 5 shots
+
+	=====   LEVEL 5 REQUIREMENT   =====
+	
+Polymer-Alloy Environment Suit  $1200
+Suit integrity: 10 shots
+
+	=====   LEVEL 10 REQUIREMENT   =====
+
+Kevlar Fiber Environment Suit  $2000
+Suit integrity: 20 shots
+
 
     =====================================
 	  STANDARD BALLISTIC ARMOURS
     =====================================
-Adding any armor materials to a chest, leg, or arm rig
-reduces your movement speed by -1m per material added.
+Adding any armor materials to a chest, leg, or arm rig adds to
+your movement speed penalties by -1m per material added.
 

```

Comment:
Should we maybe mention here that movement penalties can be remediated by stats? Or do you feel that is covered well somewhere else?

Reply:
I think that's a good idea. The more we mention it, the easier it is to remember